tasks/data.js

```diff
@@ -468,21 +474,28 @@ User.sync().then(() => {
 });
 `
 
+const envExample = `
+TEST_DATABASE_URL="postgres://localhost:5432/dbnameTest"
+DATABASE_URL="postgres://localhost:5432/dbname"
```

Comment:
Could we use an example such as `'postgres://user:pass@example.com:5432/dbnameTest'` here. 
Those users which database have fewer privileges might face issues.